src/style.css

```diff
@@ -151,3 +151,27 @@ a {
 .token.variable {
   color: #b56000 !important; /* #e90; */
 }
+
+/* Restore focus outlines to the wrapper elements around RSelect boxes */
+[role="combobox"]:focus-within {
+  outline: -webkit-focus-ring-color auto 1px;
+}
```

Comment:
For some inexplicable reason, `react-select` explicitly hides the focus ring from combo boxes. Keyboard users have no idea where they are... :(